libc/include/llvm-libc-macros/linux/CMakeLists.txt

```diff
@@ -16,12 +16,6 @@ add_header(
     sys-ioctl-macros.h
 )
 
-add_header(
```

Comment:
does this also need to be removed from the cmakelists for the other sys-mman-macros.h?

Reply:
I suppose it is automatically detected inside `add_macro_header`:
```cmake
if(TARGET libc.include.llvm-libc-macros.${LIBC_TARGET_OS}.${name})
    add_header(
      ${name}
      HDR
        ${MACRO_HEADER_HDR}
      DEPENDS
        .${LIBC_TARGET_OS}.${name}
        ${MACRO_HEADER_DEPENDS}
    )
  else()
    add_header(
      ${name}
      HDR
        ${MACRO_HEADER_HDR}
      DEPENDS
        ${MACRO_HEADER_DEPENDS}
    )
  endif()
```